datastore/datastore.go

```diff
@@ -5,9 +5,9 @@ import (
 	"k8s.io/client-go/informers"
 	clientset "k8s.io/client-go/kubernetes"
 	appslisters "k8s.io/client-go/listers/apps/v1"
-	batchlisters_v1beta1 "k8s.io/client-go/listers/batch/v1beta1"
+	batchlisters_v1 "k8s.io/client-go/listers/batch/v1"
 	corelisters "k8s.io/client-go/listers/core/v1"
-	policylisters "k8s.io/client-go/listers/policy/v1beta1"
+	policylisters "k8s.io/client-go/listers/policy/v1"
```

Comment:
NIT: for group discussion any reason we include version info in the naming for some of these but not others?

Reply:
Good point! I agree that we should standardize it be `name+version` in other ticket

Reply:
Let's create a ticket for this refactoring.